pkgs/by-name/fl/flashprog/package.nix

```
@@ -0,0 +1,53 @@
+{ fetchurl
+, stdenv
+, installShellFiles
+, lib
+, libftdi1
+, libgpiod
+, libjaylink
+, libusb1
+, pciutils
+, pkg-config
+, jlinkSupport ? false
+, gpioSupport ? false
```

Why not set them as true by default?

Also, let's follow https://github.com/NixOS/rfcs/pull/169 a bit:

```suggestion
, with_jlink ? true
, with_gpio ? true
```